SeoPresentationInterface.php

```diff
@@ -23,6 +24,16 @@
 interface SeoPresentationInterface
 {
     /**
+     * Gets the SeoMetadata based on the content that contains the content.
+     *
+     * @param object $content
+     *
+     * @throws Exception\InvalidArgumentException
+     * @return SeoMetadata
+     */
+    public function getSeoMetadata($content);
```

Comment:
maybe we should just have the phpcr sitemap provider depend on the concrete implementation instead of adding this to the interface.